### PR TITLE
Ginkgo v2

### PR DESCRIPTION
**Pull request by geofffranks**
* A short explanation of the proposed change:

* An explanation of the use cases your change solves

* Instructions to functionally test the behavior change using operator interfaces (BOSH manifest, logs, curl, and metrics)

* Expected result after the change

* Current result before the change

* Links to any other associated PRs

* [ ] I have viewed signed and have submitted the Contributor License Agreement

* [ ] I have made this pull request to the `develop` branch

* [ ] I have run all the unit tests using `scripts/run-unit-tests-in-docker`

* [ ] (Optional) I have run Routing Acceptance Tests and Routing Smoke Tests on bosh lite

* [ ] (Optional) I have run CF Acceptance Tests on bosh lite

### CODE REVIEW

File: src/code.cloudfoundry.org/go.mod [hunk 1]
```diff
@@ -63,20 +63,16 @@ require (
...
 
 require (
-	code.cloudfoundry.org/consuladapter v0.0.0-20210615194356-31457193d2fd // indirect
 	code.cloudfoundry.org/diego-logging-client v0.0.0-20210622170659-8861ae5ba2ed // indirect
 	code.cloudfoundry.org/durationjson v0.0.0-20210615172401-3a89d41c90da // indirect
 	code.cloudfoundry.org/go-diodes v0.0.0-20190809170250-f77fb823c7ee // indirect
 	code.cloudfoundry.org/go-loggregator/v8 v8.0.5 // indirect
 	code.cloudfoundry.org/inigo v0.0.0-20210615140442-4bdc4f6e44d5 // indirect
-	code.cloudfoundry.org/rep v0.1441.2 // indirect
+	code.cloudfoundry.org/lager v1.1.1-0.20230321195817-3d52f427a2d2 // indirect
```
Comment (dsabeti) [inline src/code.cloudfoundry.org/go.mod]: @geofffranks I think this PR might be premature needs one more change. 

There's still a reference to lager v1. It took a bit of digging, but the only reference I could find to `code.cloudfoundry.org/lager` without a `v3` namespace was in the [BBS packages that get vendored in](https://github.com/cloudfoundry/routing-release/blob/ginkgo-v2/src/code.cloudfoundry.org/vendor/code.cloudfoundry.org/bbs/models/task.go#L10).

Luckily, all it took was a quick update to BBS to get rid of the reference to lager v1:
```
go get code.cloudfoundry.org/bbs
go mod tidy
go mod vendor
```

I don't have push access to this branch, so I'll let you fix it.

File: scripts/run-unit-tests-in-docker [hunk 2]
```diff
@@ -6,7 +6,7 @@ ROUTING_RELEASE_DIR="${BASEDIR}/.."
 # COMPONENT takes only single package name
 COMPONENT=$1
 PACKAGE=$2
-IMAGE="cloudfoundry/cf-routing-pipeline"
+IMAGE="test-1.19"
```
Comment (dsabeti) [inline scripts/run-unit-tests-in-docker]: Was this included accidentally? I couldn't run the tests on my machine and had to change it back to cloudfoundry/cf-routing-pipeline.

**Review by dsabeti — CHANGES_REQUESTED** [review]
Made a few comments in-line to suggest changes

File: scripts/run-unit-tests-in-docker (hunk 2 above)
Comment (geofffranks) [reply scripts/run-unit-tests-in-docker]: Yep. Fixed.

File: src/code.cloudfoundry.org/go.mod (hunk 1 above)
Comment (geofffranks) [reply src/code.cloudfoundry.org/go.mod]: Fixed

**Comment by geofffranks** [comment]
Fixed the issues as well as the merge conflicts

**Review by dsabeti — APPROVED** [review]

**Pull request merged**